mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -82,16 +81,19 @@ namespace mbf_abstract_nav
 
     /**
      * @brief Constructor
-     * @param condition Thread sleep condition variable, to wake up connected threads
-     * @param controller_plugin_type The plugin type associated with the plugin to load
-     * @param tf_listener_ptr Shared pointer to a common tf listener
+     * @param name Name of this execution
+     * @param controller_ptr Pointer to the controller plugin
+     * @param robot_info Current robot state
+     * @param vel_pub Velocity publisher
+     * @param goal_pub Current goal publisher
+     * @param config Initial configuration for this execution
      */
     AbstractControllerExecution(
         const std::string &name,
         const mbf_abstract_core::AbstractController::Ptr &controller_ptr,
+        const mbf_utility::RobotInformation &robot_info,
```

Comment:
I think this one is missing in the includes

Reply:
ah... shit, I forgot to update the tests :see_no_evil: